docs/code-samples/settlement/daml/Scripts/Setup.daml

```diff
@@ -0,0 +1,188 @@
+    mapA createParty ["CentralBank", "Bank", "Bank2", "Alice", "Bob", "Charlie", "Requestor"]
+
+  -- Account Factory (used to create accounts @Bank1).
+  accountFactoryCid <- toInterfaceContractId @Account.F <$> submit bank1 do
+    createCmd Account.Factory with provider = bank1; observers = M.empty
+
+  -- Holding Factory (used to create holdings with the desired implementation @Bank1).
+  holdingFactoryCid <- toInterfaceContractId @Holding.F <$> submit bank1 do
+    createCmd Fungible.Factory with
+      provider = bank1
+      observers = M.fromList [("DisclosedToAll", S.fromList all)]
+
+  -- Set up accounts @Bank1.
+  let controllersWithOwner = toControllers accountControllers bank1
+  aliceRequestCid <- submit alice do
+    createCmd CreateAccount.Request with
+      custodian = bank1
+      owner = alice
+      accountControllers = controllersWithOwner alice
+  aliceAccount <- submit bank1 do
+    exerciseCmd aliceRequestCid CreateAccount.Accept with
+      label = "Alice@Bank1"
+      description = "Account of Alice at Bank1"
+      accountFactoryCid
+      holdingFactoryCid
+      observers = [bob, charlie] -- disclosing to all clients of Bank1
+  bobRequestCid <- submit bob do
+    createCmd CreateAccount.Request with
+      custodian = bank1
+      owner = bob
+      accountControllers = controllersWithOwner bob
+  bobAccount <- submit bank1 do
+    exerciseCmd bobRequestCid CreateAccount.Accept with
+      label = "Bob@Bank1"
+      description = "Account of Bob at Bank1"
+      accountFactoryCid
+      holdingFactoryCid
+      observers = [alice, charlie] -- disclosing to all clients of Bank1
+  charlieRequestCid <- submit charlie do
+    createCmd CreateAccount.Request with
+      custodian = bank1
+      owner = charlie
+      accountControllers = controllersWithOwner charlie
+  charlieAccount <- submit bank1 do
+    exerciseCmd charlieRequestCid CreateAccount.Accept with
+      label = "Charlie@Bank1"
+      description = "Account of Charlie at Bank1"
+      accountFactoryCid
+      holdingFactoryCid
+      observers = [alice, bob]  -- disclosing to all clients of Bank1
+
+  -- Central bank1 creates the cash instrument.
+  now <- getTime
+  let
+    cashInstrument = Instrument with
+      depository = cb
+      issuer = cb
+      id = Id "USD"
+      version = "0"
+      description = "Instrument representing units of USD"
+      validAsOf = now
+      observers = M.empty
+    instrument = Instrument.getKey $ toInterface cashInstrument
+  cashInstrumentCid <- toInterfaceContractId @Instrument.I <$> submit cb do
+    createCmd cashInstrument
+
+  -- Alice deposits cash at the bank1.
+  aliceRequestCid <- submit alice do
+    createCmd CreditAccount.Request with
+      account = aliceAccount
+      instrument
+      amount = 1000.0
+  aliceHoldingCid <- fromInterfaceContractId @Transferable.I <$> submit bank1 do
+    exerciseCmd aliceRequestCid CreditAccount.Accept
+
+  pure Setup with ..
```

Comment:
`..` notation should not be used according to style guide

Reply:
I made it explicit.